node/src/components/consensus/era_supervisor.rs

```diff
@@ -204,15 +209,25 @@ where
 
         // Asynchronously collect the information needed to initialize all recent eras.
         let effects = async move {
-            info!(?era_ids, "collecting key blocks");
+            info!(?era_ids, "collecting key blocks and booking blocks");
+
             let key_blocks = effect_builder
                 .collect_key_blocks(era_ids.iter().cloned())
                 .await
                 .expect("should have all the key blocks in storage");
 
+            let booking_blocks = collect_booking_block_hashes(
+                effect_builder,
+                era_ids.clone(),
+                auction_delay,
+                last_activation_point,
+            )
+            .await;
+
             if current_era > activation_era_id + bonded_eras * 2 {
                 // All eras can be initialized using the key blocks only.
```

Comment:
```suggestion
```

Reply:
Deleted in [`45e2470` (#1068)](https://github.com/CasperLabs/casper-node/pull/1068/commits/45e247049e02d72cbd9c9044cf0e7535f192b9fe)